.flowconfig

```diff
@@ -0,0 +1,8 @@
+[ignore]
+.*/node_modules/documentation/*
+
+[libs]
+
+[include]
+
+[options]
```

Comment:
@vmx guessing that these .flowconfig files will be removed in your effort to add flow directly to aegir, correct?

Reply:
I don't believe that would be possible given that flow uses .flowconfig to detect the root what needs to be type-checked.

Reply:
Huh, is there no way for us to specify where the config file is located? Feels very suboptimal if we have to add this to every repository...

Reply:
> Huh, is there no way for us to specify where the config file is located? Feels very suboptimal if we have to add this to every repository...

As far as I know it's still the case (at least there is no documented parameter to a flow command). Please note that it's not as much of an issue about specifying a config (`.flowconfig`  could be a blank file) but rather about not being able to specify a root from which to type-check.

Reply:
@VictorBjelkholm I hope to have it go away. @garrensmith will have a look.

Reply:
Reading [here](https://flow.org/en/docs/config/), it doesn't seem like we can remove the flowconfig file. It should be a very simple file for each project. I don't think it would be to much of an issue having it in each repo. We could maybe look at having AEgir generate a generic .flowconfig file for a repo.